src/components/TransactionFlow/TxReceipt.tsx

```diff
@@ -166,21 +166,21 @@ export default function TxReceipt({
     }
   });
 
-  const assetRate = useCallback(() => {
+  const assetRate = (() => {
     if (displayTxReceipt && path(['asset'], displayTxReceipt)) {
       return getAssetRate(displayTxReceipt.asset);
     } else {
       return getAssetRate(txConfig.asset);
     }
-  }, [displayTxReceipt, txConfig.asset]);
+  })();
 
-  const baseAssetRate = useCallback(() => {
+  const baseAssetRate = (() => {
     if (displayTxReceipt && path(['baseAsset'], displayTxReceipt)) {
       return getAssetRate(displayTxReceipt.baseAsset);
     } else {
       return getAssetRate(txConfig.baseAsset);
     }
-  }, [displayTxReceipt, txConfig.baseAsset]);
+  })();
```

Comment:
Do we really need an IIFE? Can we create a tested helper function instead?

Reply:
It's just not reusable, so I didnt feel like it made sense to abstract out?

Reply:
ok